src/api/objects/MutableDomainObject.js

```diff
@@ -113,6 +113,9 @@ class MutableDomainObject {
     }
     $destroy() {
         this._observers.forEach(observer => observer());
+        //TODO: the following deletes will not work since they're not configurable properties.
+        // Also, this._globalEventEmitter doesn't seem to change after the above observer listeners are 'removed'
+        // Also, probably need to pop items off this._observers here or the items will remain on the array.
```

Comment:
I believe that modern JS VMs use reachability to determine if objects can be GC'd, so even if the array holds references to objects, if nothing references the array itself then it will be garbage collected.

Reply:
OK, that's fine too. We can leave them in the array.

Reply:
See below for resolution of this comment.